### PR TITLE
update LDAP doc

### PR DESCRIPTION
**Pull request by thomasmckay**
- broke up screenshot into sections
- described each section
- include config.yaml sample for each section

### CODE REVIEW

File: modules/proc_manage-ldap-setup.adoc [hunk 1]
```diff
@@ -32,43 +32,110 @@ panel *_as the user that was just created._*
...
+=== UID and Mail attributes
+
+image:authentication-ldap-uid-mail.png[UID and Mail]
+
+* The UID attribute is the name of the property field in LDAP user record to use as the *username*. Typically "uid".
+* The Mail attribute is the name of the property field in LDAP user record that stores user e-mail address(es). Typically "mail".
+* Either of these may be used during login.
+* The logged in username must exist in User Relative DN.
+* _sAMAccountName_ is the UID attribute for against Microsoft Active Directory setups.
+* _config.yaml_
```
[the remark this answers is not in the document]
Comment (thomasmckay) [reply modules/proc_manage-ldap-setup.adoc]: For the multi-line entries I changed it to "Here is an example of the resulting entries in the config.yaml file" (it's just a singular example with multiple entries).

File: modules/proc_manage-ldap-setup.adoc [hunk 2]
```diff
@@ -32,43 +32,110 @@ panel *_as the user that was just created._*
 [[setup-ldap-configuration]]
 == Setup LDAP Configuration
```
[the remark this answers is not in the document]
Comment (thomasmckay) [reply modules/proc_manage-ldap-setup.adoc:34]: done!

File: modules/proc_manage-ldap-setup.adoc [hunk 3]
```diff
@@ -32,43 +32,110 @@ panel *_as the user that was just created._*
...
-ou=Users,ou=NYC and ou=Users,ou=SFO). Simply type in the Organizational
-Units and click on Add button to add multiple RDNs
-* sAMAccountName is the UID attribute for against Microsoft Active
-Directory setups
-* {productname} searches "User Relative DN" with subtree scope. For
-example, if your Organization has Organizational Units NYC and SFO under
-the Users OU (`ou=SFO,ou=Users and ou=NYC,ou=Users`), {productname} 
-can authenticate users from both the NYC and SFO Organizational Units if
-the User Relative DN is set to Users (ou=Users)
+In the config tool locate the Authentication section and select “LDAP” from the drop-down menu. Update LDAP configuration fields as required.
```
[the remark this answers is not in the document]
Comment (thomasmckay) [reply modules/proc_manage-ldap-setup.adoc]: done

**Comment by chrisnegus** [comment]
LGTM

File: modules/proc_manage-ldap-setup.adoc [hunk 4]
```diff
@@ -30,45 +30,112 @@ created. Restart the {productname} container and log in to the Superuser
...
+AUTHENTICATION_TYPE: LDAP
+....
+
+=== Full LDAP URI
+
+image:authentication-ldap-uri.png[LDAP server URI]
+image:authentication-ldap-ssl.png[LDAP server SSL]
+
+* The full LDAP URI, including the _ldap://_ or _ldaps://_ prefix.
+* A URI beginning with _ldaps://_ will make use of the provided SSL certificate(s) for TLS setup.
```
Comment (LiZhang19817) [inline modules/proc_manage-ldap-setup.adoc:50]: there're two locations on Quay-config app to upload customized certificate, need to let customers know which one is valid for LDAPS.

**Pull request merged**